app/layouts/DashboardLayout/BalanceIndicator.view/BalanceIndicator.view.tsx

```diff
@@ -1,25 +1,51 @@
 import React from 'react';
 import type { FC } from 'react';
 
-import { Box, Button, makeStyles, Typography, useMediaQuery } from '@material-ui/core';
+import {
+  Box,
+  Button,
+  makeStyles,
+  Typography,
+  useMediaQuery,
+  Select,
+  MenuItem,
+  ListItemIcon,
+} from '@material-ui/core';
+import MonetizationOnOutlinedIcon from '@material-ui/icons/MonetizationOn';
 import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
 
 import { MOBNumberFormat } from '../../../components';
 import { MOBIcon } from '../../../components/icons';
+import { TokenIds } from '../../../constants/app';
 import { GOLD_LIGHT } from '../../../constants/colors';
+import { ReduxStoreState } from '../../../redux/reducers/reducers';
+import { setTokenId } from '../../../redux/services';
 import { Theme } from '../../../theme';
 import { BalanceIndicatorProps } from './BalanceIndicator';
 
 const useStyles = makeStyles((theme: Theme) => ({
+  formControlLabel: {
+    left: 24,
+  },
   icon: {
-    height: 25,
-    paddingRight: 5,
-    width: 25,
+    left: 0,
+  },
+  iconElement: {
+    height: 24,
+    width: 24,
+  },
+  iconOpen: {
+    transform: 'none',
   },
   item: {
     padding: theme.spacing(3, 0, 0, 0),
     textAlign: 'center',
   },
+  selectSelect: {
+    paddingLeft: '24px',
+    paddingRight: '8px !important',
```

Comment:
using `!important` is a css antipattern, is there an easy way around this? NBD if not, but its usually an indication of hard-to-parse css inheritance structures

Reply:
I agree, but I wasn't sure what was overwriting the style and given that this might not be the final styling, I didn't want to spend the time figuring it out